script_library/pre-flight-checks.sh

```diff
@@ -1,5 +1,18 @@
 #!/usr/bin/env bash
 
+check_common_env_vars_set (){
+    # basic checks that are needed for everything else!
+    if [ -z ${SOCOK8S_ENVNAME+x} ]; then
+        echo "No SOCOK8S_ENVNAME given. export SOCOK8S_ENVNAME=... for setting a env name" && exit 1
+    fi
+    # NOTE(toabctl): SOCOK8S_WORKSPACE_BASEDIR is always set in run.sh
+    if [ -z ${SOCOK8S_WORKSPACE_BASEDIR+x} ]; then
```

Comment:
If it's always set, why do we check for it being set?

Reply:
hm. good question. I can remove the check but it feels safer to me when I leave it in. Should I remove it?

Reply:
Yeah.

Reply:
I would.